AUTHORS

```diff
@@ -494,3 +494,4 @@ a license to everyone to use it as detailed in LICENSE.)
 * Mathias Westerdahl <mwesterdahl76@gmail.com>
 * Philip Rideout <prideout@google.com> (copyright owned by Google, LLC)
 * Shrek Shao (shrekshao@google.com) (copyright owned by Google, LLC)
+* Sean Maher <seanptmaher@gmail.com> (copyright owned by Google, LLC)
```

Comment:
This reminds me that similar language about Google owning the copyright should probably go in the relevant SIMDe licenses...

Reply:
Hm, you're right. I'll see if I can put together a PR for that.